GiEnJul/ClientApp/src/common/functions/IsMobile.tsx

```diff
@@ -0,0 +1,3 @@
+export const isMobile = () => {
+    return window.innerWidth < 576;
```

Comment:
this maybe cause bugs as the users innerWidth can exceed 576 in landscape mode for many devices.
a more thorough check is necessary. Look into MDN for ways to check if a device is mobile device :)